main.tex

```diff
@@ -482,12 +482,14 @@ \section*{Software design}
     model. The viewing angle, camera perspective, and other details can be
     manipulated with a mouse or from the toolbar above the window. Tabs on the
     right provide sliders and text inputs for all 21 configuration variables
-    and allow the user to interactively set the configuration. The ``Reset
-    Configuration'' button sets all the configuration variables to zero. The
-    mass center and inertia ellipsoid for the entire body can be toggled with
-    the checkboxes. Finally, the whole-body mass, center of mass coordinates,
-    and inertia tensor are displayed in the bottom right and are updated
-    interactively as the configuration is altered.
+    and allow the user to interactively set the configuration. The
+    configuration limits prevent extreme configurations, and do not reflect the
```

Comment:
Some people can flex their bodies beyond our limits and my not be extreme for them. Here is one rewrite:

The configuration limits prevent likely extreme configurations and do not necessarily reflect the range of motion of the subject.

And I'm not sure if I like "extreme". "Abnormal" may be better.

The configuration limits prevent abnormal configurations and do not necessarily reflect the range of motion of the subject.

Take what you want from the above, if any, and merge.